website/content/docs/connect/proxies/envoy.mdx

```diff
@@ -10,7 +10,7 @@ Consul Connect has first class support for using
 [Envoy](https://www.envoyproxy.io) as a proxy. Consul configures Envoy by
 optionally exposing a gRPC service on the local agent that serves [Envoy's xDS
 configuration
-API](https://www.envoyproxy.io/docs/envoy/latest/api-docs/xds_protocol).
+API](https://www.envoyproxy.io/docs/envoy/v1.17.2/api-docs/xds_protocol).
```

Comment:
I pinned all envoy docs to `v1.17.2` since that's the last version that has docs for xDS v2 and xDS v3 which seemed handy if you had to do something like reference both while upgrading.